config/locales/en.yml

```diff
@@ -93,6 +93,14 @@ en:
     cats: "Cats"
     browse: "Browse"
     view: "View"
+    about_us: "About us"
+    adopt: "Adopt"
+    become_adopter_now: "Become an adopter now!"
+    how_to_apply: "How to apply for adoption"
+    step_1: "1. Create Account"
+    step_2: "2. Complete Questionnaire"
+    step_3: "3. Browse Pets"
+    step_4: "4. Click Apply"
```

Comment:
Just a nit, but I'm thinking it would be a little more organized to scope these like  `en.organizations.home.index`. "About us" could probably stay in default as it's used on the root page as well (needs to be updated in a separate issue)

If you change to this scope, this will allow you to use rails i18n lazy lookup.

`t(".how_to_apply"` will look for `en.organizations.home.index.how_to_apply`

Reply:
I like it, yeah! I was wondering whether to have it in general or not tbh. Will refactor it today!